app/controllers/concerns/application_shared.rb

```diff
@@ -15,4 +15,12 @@ def set_timezone
     # Reset timezone for the next thread
     Time.zone = default_timezone
   end
-end
\ No newline at end of file
+
+  private
+
```

Comment:
@domcleal Tried that but it won't help. The block is evaluated when included on ApplicationController and BaseController, so the only _params methods defined are application_params and base_params. 
```
  included do
    define_method "#{controller_name.singularize}_params" do
      params.require(controller_name.singularize.to_sym).
        permit(*resource_class.permitted_attributes)
    end
  end
```